model/clusters_mgmt/v1/aws_machine_pool_type.model

```diff
@@ -27,4 +27,14 @@ class AWSMachinePool {
 
     // Associates nodepool availability zones with zone types (e.g. wavelength, local).
     AvailabilityZoneTypes [String]String
+
+    // Optional keys and values that the machine pool provisioner will add as AWS tags to all AWS resources it creates.
+    //
+    // AWS tags must conform to the following standards:
+    // - Each resource may have a maximum of 50 tags
```

Comment:
```suggestion
    // - Each resource may have a maximum of 25 tags
```

There's a soft limit right? AWS itself is 50 though

Reply:
oh right, was going off the AWS guidelines and forgot we had a smaller limit. I'll change that rn